pyserini/search/__main__.py

```diff
@@ -185,6 +187,16 @@ def define_search_args(parser):
     if args.dismax:
         query_generator = JDisjunctionMaxQueryGenerator(args.tiebreaker)
         print(f'Using dismax query generator with tiebreaker={args.tiebreaker}')
+    
+    if args.tokenizer != None:
+        analyzer = JWhiteSpaceAnalyzer()
+        searcher.set_analyzer(analyzer)
+        print(f'Using whitespace analyzer because of pretokenized topics')
+        if ('bert' in args.tokenizer):
+            tokenizer = BertTokenizer.from_pretrained('bert-base-uncased')
```

Comment:
actually, let the arg be the actual name of tokenizer?

e.g., `--tokenizer bert-base-uncased`